v03_pipeline/lib/tasks/write_remapped_and_subsetted_callset.py

```diff
@@ -36,7 +37,17 @@ class WriteRemappedAndSubsettedCallsetTask(BaseWriteTask):
     project_pedigree_path = luigi.Parameter()
 
     def complete(self) -> luigi.Target:
-        return not self.force and super().complete()
+        return (
+            not self.force
+            and super().complete()
+            and hl.eval(
+                hl.read_matrix_table(self.output().path).globals.remap_pedigree_hash
```

Comment:
did you add a test for when this is not the case?

Reply:
I think it might be covered by something that exists, but I should add an explicit one for sure!